packages/shared/src/utils/currency/currency.ts

```diff
@@ -1,6 +1,28 @@
 import { getPropertyValue, deepFreeze } from '../object';
 
-let currencies_config = {};
+export type TCurrenciesConfig = {
+    [k: string]: {
```

Comment:
```suggestion
    [key: string]: {
```
@niloo-fs it's more about who likes what of course but to be honest, to me 'key' seems to be more readable than 'k'🙂

Reply:
Yes, agree with you, will change them